witchcraft/internal/middleware/request_test.go

```diff
@@ -102,6 +105,32 @@ func TestCombinedMiddleware(t *testing.T) {
 	// start server
 	server := httptest.NewServer(r)
 	defer server.Close()
+	defer func() {
+		matcher := objmatcher.MapMatcher{
+			"type": objmatcher.NewEqualsMatcher("trace.1"),
+			"time": objmatcher.NewRegExpMatcher(".+"),
+			"span": objmatcher.MapMatcher(map[string]objmatcher.Matcher{
+				"name":      objmatcher.NewEqualsMatcher("witchcraft-go-server request middleware"),
+				"traceId":   objmatcher.NewRegExpMatcher("[a-f0-9]+"),
+				"id":        objmatcher.NewRegExpMatcher("[a-f0-9]+"),
+				"timestamp": objmatcher.NewAnyMatcher(),
+				"duration":  objmatcher.NewAnyMatcher(),
+				"tags": objmatcher.MapMatcher(
+					map[string]objmatcher.Matcher{
+						"http.status_code": objmatcher.NewEqualsMatcher("200"),
+						"http.method": objmatcher.NewEqualsMatcher("GET"),
+						"http.url": objmatcher.NewEqualsMatcher("/"),
+						"http.useragent": objmatcher.NewRegExpMatcher(".*"),
+						"http.url_details.path": objmatcher.NewEqualsMatcher("/"),
+					},
+				),
+			}),
+		}
+		entries, err := logreader.EntriesFromContent(trcOutput.Bytes())
+		assert.NoError(t, err, "unexpected error when unmarshalling trace output")
+		assert.NoError(t, matcher.Matches(map[string]interface{}(entries[0])), "unexpected content in trace output")
+	}()
+
```

Comment:
Previously there was no test that checked the expected trace output. This checks for a single span with expected name and tags.